scripts/mkcloud

```diff
@@ -151,6 +151,7 @@ function show_environment()
     echo "-------------------------------"
     echo " cloudsource: $cloudsource"
     echo "    TESTHEAD: $TESTHEAD"
+    echo "    scenario: $scenario"
```

Comment:
For me `scenario` is fine here regarding `cct`, I'm rather wondering what a scenario means here. Is it a barclamp configuration?